### ISSUE TITLE
Return to Stock

### ISSUE THREAD

**Issue by majesty92** [lukasz1992/asuswrt-xiaomi#80]
Hello Mr Lukasz 
I installed your firmware with your method when I had padavan and at the moment I would like to know how to go back to stock xiaomi firmware since I do not have breed installed? Could you please help with this issue?

**Comment by lukasz1992** [comment]
Hi, I have not done it.

Do you have this firmware and can send a link here?
If it is easy I will write a guide, but now I can only tell how to upload OpenWRT

**Comment by majesty92** [comment]
Thanks for your reply below the link for stock firmware 

https://4pda.ru/forum/dl/post/20902495/miwifi_rm2100_all_fb720_2.0.23.zip 
I would really appreciate if you could help with this

**Comment by lukasz1992** [comment]
hm... is there instruction to flash OEM from Padavan? If yes, maybe flashing Padavan and then flashing OEM is easier?
I can not download file

**Comment by majesty92** [comment]
Yes there is but I have your firmware that is awesome and I want to also flash it for my friend but If he doesn't like it I do not know how to flash padavan/ or OEM from your firmware

**Comment by lukasz1992** [comment]
OK, So please flash Padavan firmware:

1. upload it to router into /tmp directory and name it firmware.bin
2. run these commands
```mtd-write -i /tmp/firmware.bin -d linux
reboot -f```

Note: I can not guarantee it works, in the worst case you will need UART access to u-boot

**Comment by majesty92** [comment]
Thanks for your help found a way via miwifirepair tool Now I can easily go back again and clean flash your firmware, please keep up the good work it helps us a lot
